storage/innobase/log/log0recv.cc

```diff
@@ -1051,6 +1055,7 @@ static dberr_t recv_log_recover_pre_8_0_30(log_t &log) {
   return log_start(log, checkpoint_lsn, checkpoint_lsn, nullptr);
 }
 
+#ifndef XTRABACKUP
```

Comment:
ah, you are moving Log_checkpoint_location to new place.
The dilemma is whether to add #ifdef XTRABACKUP around the new place.

I think, we shouldn't add the #define. (see above)

Reply:
removed the ifdef

Reply:
@rahulmalik87 I see that you left it at its original place in storage/innobase/include/log0recv.h. Is that the intended outcome?

Reply:
no invalid ifdef present in storage/innobase/include/log0recv.h.